neo4j-to-neptune/docs/conversion-config.md

```diff
@@ -1,63 +1,59 @@
-# Label Mapping and Record Filtering
+# Conversion Configuration
 
-The `convert-csv` utility supports mapping vertex and edge labels and filtering records during the conversion process from Neo4j to Neptune format. This feature allows you to rename labels to follow different naming conventions, resolve naming conflicts, and exclude specific vertices and edges from the conversion.
+The `convert-csv` utility supports advanced configuration through YAML files that enable label mapping, ID transformation and data filtering during the Neo4j to Neptune conversion process.
 
 ## Usage
 
-Use the `--conversion-config` parameter to specify a YAML file containing the label mappings and filtering rules:
+Use the `--conversion-config` parameter to specify a YAML file containing the label mapping, ID transformation and filtering rules:
 
 ```bash
 java -jar neo4j-to-neptune.jar convert-csv \
-  -i /tmp/neo4j-export.csv \
-  -d output \
-  --conversion-config  config.yaml \
-  --infer-types
+  -i /path/to/neo4j-export.csv \
+  -d /path/to/output \
+  --conversion-config conversion-config.yaml \
+  [other options]
 ```
 
-## YAML Configuration Format
+## Configuration File Structure
 
 The configuration file should be in YAML format, using camelCase, with sections for label mapping and record filtering:
 
 ```yaml
-# Vertex label mappings
+# Label Mappings
 vertexLabels:
   OldVertexLabel: NewVertexLabel
   Person: Individual
   Company: Organization
   Product: Item
 
-# Edge label mappings  
 edgeLabels:
-  OLD_RELATIONSHIP_TYPE: NEW_RELATIONSHIP_TYPE
+  OLD_RELATIONSHIP: NEW_RELATIONSHIP
   WORKS_FOR: EMPLOYED_BY
   LIVES_IN: RESIDES_IN
   KNOWS: CONNECTED_TO
 
-# Skip vertices configuration
+# ID Transformations
+vertexIdTransformation:
+  ~id: "{template}"
```

Comment:
Nit: Might be nice to show an example id template here with some typical combination of a label and property value.